--- chirp-1/setup.py	2015-03-05 09:00:18.000000000 +0100
+++ chirp-1/setup-patched.py	2017-08-16 14:21:46.867048797 +0200
@@ -115,7 +115,7 @@
 
     locale_files = []
     for f in _locale_files:
-        locale_files.append(("share/chirp/%s" % os.path.dirname(f), [f]))
+        locale_files.append(("share/%s" % os.path.dirname(f), [f]))
 
     print "LOC: %s" % str(locale_files)
 
@@ -129,8 +129,8 @@
         data_files=[('share/applications', desktop_files),
                     ('share/chirp/images', image_files),
                     ('share/chirp', xsd_files),
-                    ('share/doc/chirp', ['COPYING']),
-                    ('share/pixmaps', ['share/chirp.png']),
+                    ('share/licenses/chirp', ['COPYING']),
+                    ('share/icons/hicolor/scalable/apps', ['share/chirp.png']),
                     ('share/man/man1', ["share/chirpw.1"]),
                     ('share/chirp/stock_configs', stock_configs),
                     ] + locale_files)
--- chirp-1/chirp/platform.py	2017-02-07 09:01:12.000000000 +0100
+++ chirp-1/chirp/platform-patched.py	2017-08-16 14:12:55.212808274 +0200
@@ -249,6 +249,7 @@
         share_candidates = [
             os.path.join(execpath, "share"),
             os.path.join(sys.prefix, "share"),
+            "/app/share",
             "/usr/local/share",
             "/usr/share",
         ]
