--- setup.py	2017-08-16 14:56:21.862579188 +0200
+++ setup-patched.py	2017-08-17 15:10:31.909810324 +0200
@@ -115,7 +115,7 @@
 
     locale_files = []
     for f in _locale_files:
-        locale_files.append(("share/chirp/%s" % os.path.dirname(f), [f]))
+        locale_files.append(("share/chirp/%s" % os.path.dirname(f), [f]))
 
     print "LOC: %s" % str(locale_files)
 
@@ -129,8 +129,8 @@
         data_files=[('share/applications', desktop_files),
                     ('share/chirp/images', image_files),
                     ('share/chirp', xsd_files),
-                    ('share/doc/chirp', ['COPYING']),
-                    ('share/pixmaps', ['share/chirp.png']),
+                    ('share/licenses/chirp', ['COPYING']),
+                    ('share/pixmaps', ['share/chirp.png']),
                     ('share/man/man1', ["share/chirpw.1"]),
                     ('share/chirp/stock_configs', stock_configs),
                     ] + locale_files)
@@ -144,7 +144,7 @@
           version="0.3",
           scripts=["rpttool"],
           description="A frequency tool for ICOM D-STAR Repeaters",
-          data_files=[('/usr/sbin', ["tools/icomsio.sh"])],
+          data_files=[('/app/bin', ["tools/icomsio.sh"])],
           )
 
 
