--- chirp-1/share/chirp.desktop	2017-08-16 14:34:38.329338965 +0200
+++ chirp-1/share/chirp-patched.desktop	2017-08-16 14:34:14.983511093 +0200
@@ -1,13 +1,13 @@
 [Desktop Entry]
-Version=0.1
-Encoding=UTF-8
 Type=Application
-Exec=chirpw
-Icon=chirp.png
-StartupNotify=true
+Version=1.0
+Name=CHIRP
+GenericName=Radio Programming Tool
+Comment=Program amateur radios
+Icon=chirp
+Exec=chirpw %F
 Terminal=false
-Categories=Application;HamRadio
 MimeType=inode/directory
-Name=CHIRP
-Comment=CHIRP Radio Programming Tool
-GenericName=CHIRP Radio Programming Tool
+Categories=Utility;HamRadio
+Keywords=Hamradio;Programming;Handheld;Radio;Amateur;Programmer
+StartupNotify=true
